frontend/src/containers/PublicGroup.js

```diff
@@ -103,6 +110,8 @@ export class PublicGroup extends Component {
     this.donateToGroupRef = donateToGroup.bind(this);
     this.closeDonationFlowRef = this.closeDonationFlow.bind(this);
     this.saveNewUserRef = saveNewUser.bind(this);
+    this.saveGroupRef = saveGroup.bind(this);
+    this.cancelGroupEditsRef = onCancelGroupEdits.bind(this);
```

Comment:
(optional) You can simply override it: `this.saveGroup = saveGroup.bind(this);` so that you don't introduce a new variable